tests/release/release.go

```diff
@@ -152,13 +152,14 @@ var _ = framework.ReleaseSuiteDescribe("[HACBS-1108]test-release-service-happy-p
 				}
 
 				return len(pipelineRunList.Items) > 0 && err == nil
-			}, avgPipelineCompletionTime, defaultInterval).Should(BeTrue())
+			}, avgControllerQueryTimeout, defaultInterval).Should(BeTrue())
 
 			release, err := framework.ReleaseController.GetRelease(releaseName, devNamespace)
 			if err != nil {
 				klog.Error(err)
 			}
 			Expect(release.Status.ReleasePipelineRun == (fmt.Sprintf("%s/%s", pipelineRunList.Items[0].Namespace, pipelineRunList.Items[0].Name))).Should(BeTrue())
-		})
+			// We add the namespace deletion timeout as this is the last test so must also take into account the code in AfterAll
```

Comment:
Can you make it align with line 163? So one less tab?

Reply:
I updated however the auto formatting of my VSCode IDE returns the comments to same tab place (it believes that is the correct placement)